langchain/llms/openai.py

```diff
@@ -155,7 +155,10 @@ def _generate(
         # Includes prompt, completion, and total tokens used.
         _keys = {"completion_tokens", "prompt_tokens", "total_tokens"}
         for _prompts in sub_prompts:
-            response = self.client.create(prompt=_prompts, **params)
+            @retry()
+            def create():
+                return self.client.create(prompt=_prompts, **params)
```

Comment:
There are multiple places `client.create` is called.

Reply:
The other usage of client.create() is as a generator for the streaming case; I need to think more about how to wrap the generator for retry.  To make it fully robust to against transient failures that occur mid-stream may require retry logic within the openai client.